index.js

```diff
@@ -1253,9 +1263,14 @@ ContainerWindow.prototype = {
   _configureAllTabsMenu() {
     return this._configureMenu("alltabs_containersTab", null, e => {
       const userContextId = parseInt(e.target.getAttribute("data-usercontextid"), 10);
-      ContainerService.openTab({
-        userContextId: userContextId,
-        source: "alltabs-menu"
+      ContainerService.showTabs({
```

Comment:
Would it be easier to show all tabs in the openTab function so this would never happen again?